### PR TITLE
10.0 mig sale exception

### PR DESCRIPTION
**Pull request by mourad-ehm**
[WIP] I proposed an issue to split this module on 2 modules (see #408).

### CODE REVIEW

**Comment by mourad-ehm** [comment]
This PR depends on this one https://github.com/OCA/server-tools/pull/775  .

**Comment by mourad-ehm** [comment]
The commit  b23550c  must be removed before merging this PR.

**Comment by florian-dacosta** [comment]
Travis is still red.
Also, you should update the README file (description is a bit outdated, version number is wrong...)
Also some 'openerp' should be replaced by 'odoo' both in python file for import and in xml files

File: sale_exception/__manifest__.py [hunk 1]
```diff
@@ -4,18 +4,17 @@
 
 {'name': 'Sale Exception',
  'summary': 'Custom exceptions on sale order',
- 'version': '9.0.1.0.0',
+ 'version': '10.0.2.0.0',
  'category': 'Generic Modules/Sale',
  'author': "Akretion, Sodexis, Odoo Community Association (OCA)",
```
Comment (bealdav) [inline sale_exception/__manifest__.py]: Add Camptocamp

**Comment by mourad-ehm** [comment]
Hi @angelmoya could you review this PR. Thanks

File: sale_exception/README.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ help us smashing it by providing a detailed and welcomed `feedback
 <https://github.com/OCA/
 sale-workflow/issues/new?body=module:%20
 sale_exception%0Aversion:%20
-9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (angelmoya) [inline sale_exception/README.rst]: Best to use short version, check template on mantainer tools

File: sale_exception/__manifest__.py [hunk 3]
```diff
@@ -4,18 +4,17 @@
 
 {'name': 'Sale Exception',
  'summary': 'Custom exceptions on sale order',
- 'version': '9.0.1.0.0',
+ 'version': '10.0.2.0.0',
```
Comment (angelmoya) [inline sale_exception/__manifest__.py:7]: why 10.0.2.0.0 and not 10.0.1.0.0?

**Review by angelmoya — COMMENTED** [review]
Only few comments, I will check it runbot.

File: sale_exception/__manifest__.py (hunk 3 above)
Comment (mourad-ehm) [reply sale_exception/__manifest__.py:7]: 10.0.2.0.0  because the orgin sale_exception was  splited into 2 modules

**Comment by bealdav** [comment]
:+1:

File: .travis.yml [hunk 4]
```diff
@@ -31,6 +31,7 @@ virtualenv:
   system_site_packages: true
 
 install:
+  - git clone https://github.com/akretion/server-tools.git ${HOME}/server-tools-base_exception -b 10.0-add-base_exception
```
Comment (sebastienbeau) [inline .travis.yml]: As the main PR is merge can you remove it https://github.com/OCA/server-tools/pull/775

File: oca_dependencies.txt [hunk 5]
```diff
@@ -3,3 +3,4 @@ stock-logistics-workflow
 stock-logistics-transport
 account-closing
 server-tools
+server-tools-ak https://github.com/akretion/server-tools.git 10.0-add-base_exception
```
Comment (sebastienbeau) [inline oca_dependencies.txt]: same here

**Review by sebastienbeau — CHANGES_REQUESTED** [review]

File: oca_dependencies.txt (hunk 5 above)
Comment (mourad-ehm) [reply oca_dependencies.txt]: OK

**Comment by mourad-ehm** [comment]
Hi @sebastienbeau , I maked the change. Could you have a look please ?

**Review by sebastienbeau — APPROVED** [review]

**Comment by sylvainc** [comment]
LGTM (code review) :+1:

**Comment by pedrobaeza** [comment]
Next time, please squash all the "OCA Transbot commit" according to this guide https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by mourad-ehm** [comment]
I w'll do it.

**Pull request merged**